### ISSUE TITLE
problem authenticating - abort

### ISSUE THREAD

**Issue by agtsaavedra** [davidearl/webauthn#63]
Hello, I am trying to use the library for an implementation. I am having a problem after registering the device (it works perfectly) and then when trying to authenticate, the Windows modal appears to enter a USB access key but later when canceling (to enter with the fingerprint stored on my device) the operation aborts .
This happens both in the demo and in my server integration.
![image](https://github.com/davidearl/webauthn/assets/110622839/bee2e91e-d31e-4625-ad9d-b5779336e4eb)

I can't find where the problem is, the code was not changed at all.

**Comment by davidearl** [comment]
What are you expecting to happen if you cancel? The demo is just reporting that happened.

**Comment by agtsaavedra** [comment]
That's correct, the drawback is that I need it to give me the option to authenticate with a fingerprint, it only allows through a USB device. Normally when I hit cancel it allows me to log in with other devices (cell phone).

![image](https://github.com/davidearl/webauthn/assets/110622839/b114cc9e-7b63-4f45-90b0-d52ac256f6d3)

**Comment by hsinatnias** [comment]
The same with me also when I tried. If I use the windows login PIN in windows Hello at the time of user registration, it works for login also. But if I use an external device like an android phone at the time of registration, it doesn't give option to use android on login

**Comment by hsinatnias** [comment]
> The same with me also when I tried. If I use the windows login PIN in windows Hello at the time of user registration, it works for login also. But if I use an external device like an android phone at the time of registration, it doesn't give option to use android on login

**Comment by hsinatnias** [comment]
It seems, if we add 'hybrid' in transports under function "prepareForLogin" in file  "WebAuthn\WebAuthn.php", we could get cross-platform enabled. I am not sure if it has any security implications as I am new to webauthn.
